src/web/components/ClickToView.stories.tsx

```diff
@@ -53,6 +53,9 @@ const RoleStory = ({
 				className={css`
 					max-width: 620px;
 					clear: left;
+					strong {
+						font-weight: bold;
+					}
```

Comment:
Note to self: Look at a way to set this global css somewhere higher up in Storybook - I think there's a resets options or something similar.